Makefile

```diff
@@ -0,0 +1,554 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
Maybe you don't want to upload this file onto GitHub?

Reply:
No I don't want it, I put it in .gitignore but it is still there, I'll delete it